pkg/planner/core/plan_cache_utils.go

```diff
@@ -229,6 +229,12 @@ func GeneratePlanCacheStmtWithAST(ctx context.Context, sctx sessionctx.Context,
 	return preparedObj, p, paramCount, nil
 }
 
+// relatedTblInfo is used to hold related table for building plan cache key.
+type relatedTblInfo struct {
+	id  int64
+	rev uint64
```

Comment:
```suggestion
	ver uint64
```

the name should be `version`?

Reply:
But the value comes from `TableInfo.Revision`.